tests/contrib/test_django_rest_framework.py

```diff
@@ -261,3 +261,15 @@ class Meta:
         else:
             assert not serializer.is_valid()
             assert serializer.errors["second_money"][0] == "Ensure this value is greater than or equal to 0."
+
+    def test_no_model_serializer(self):
```

Comment:
Confirmed this test failed prior to my fix

Reply:
:pray: